geometry_msgs/PoseStamped palm_goal_pose_world
---
bool success
trajectory_msgs/JointTrajectory trajectory
